pkg/volume/csi/csi_mounter.go

```diff
@@ -245,6 +245,35 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 		}
 	}
 
+	// Save volume info in pod dir
```

Comment:
Is this just a cherry pick of https://github.com/kubernetes/kubernetes/pull/108874? Should we submit that separately?

reading the original pr that moved this to NewMounter, it sounds like at the time it was done for code cleanliness and not related to fixing the original issue: https://github.com/kubernetes/kubernetes/pull/64519

Reply:
You're right, I removed this commit.